src/components/Thumbnail/Thumbnail.driver.ts

```diff
@@ -0,0 +1,33 @@
+import {thumbnailDriverFactory as coreTuhmbnailDriverFactory, ThumbnailDriver as CoreThumbnailDriver} from 'wix-ui-core/dist/src/components/Thumbnail/Thumbnail.driver';
+import {StylableDOMUtil} from 'stylable/test-utils';
+import style from './Thumbnail.st.css';
+import {textDriverFactory, TextDriver} from '../Text/Text.driver';
+import {DriverFactory} from 'wix-ui-test-utils/driver-factory';
+
+export interface ThumbnailDriver extends CoreThumbnailDriver {
+  titleDriver: () => TextDriver; 
+  getTitle: () => string;
+  hasDescription: () => boolean;
+  getDescription: () => string;
+  hasImage: () => boolean;
+}
+
+export const thumbnailDriverFactory: DriverFactory<ThumbnailDriver> = ({element, eventTrigger}) => {
+  const coreThumbnailDriver = coreTuhmbnailDriverFactory({element, eventTrigger});
+  const stylableDOMUtil = new StylableDOMUtil(style, element);
+  const titleDriver = textDriverFactory({element: stylableDOMUtil.select('.title')});
```

Comment:
Is this the preferred way to select the element? why not use data-hook?
This creates coupling for styling and behavior

Reply:
Well I guess that the rule of thumb which we work with today is:
data-hook - For testing 
className - For styling

I just think that since the className is unique because it gets namespaced It is usable, so why should I add another way to find it. But yeah, I guess this is the way we do it today.

Reply:
The thing is that it is not only going to add more code to the component, it is also not a specific namespaced selector if I will have 
```
  const titleDriver = textDriverFactory({element: stylableDOMUtil.select('[data-hook="title"]')});
```

Are we sure this is what we want to do?

Reply:
Maybe in times where we already have className, we can use it as a selector for tests?